quickstarts/microsoft.azurestackhci/image-from-azure-marketplace/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "customLocationName": {
+            "value": "GEN-UNIQUE"
+        },
+        "location": {
+            "value": "eastus"
+        },
+        "imageName": {
+            "value": "GEN-UNIQUE"
```

Comment:
Not sure I follow how the imageName can be a unique, random value. Doesn't the image have to be a specific value?